pkg/manifest/os.go

```diff
@@ -152,6 +152,12 @@ type OS struct {
 	// OSTreeParent source spec (optional). If nil the new commit (if
 	// applicable) will have no parent
 	OSTreeParent *ostree.SourceSpec
+
+	// Enabling Bootupd runs bootupctl generate-update-metadata in the tree to
+	// transform /usr/lib/ostree-boot into a bootupd-compatible update
+	// payload. Only works with ostree-based images.
+	Bootupd bool
```

Comment:
Silly question, should this be `Bootupd` or `BootupdGenMetadata` (or something similar more expressive)? I guess I ask if it's unambiguous enough (and it's a honest question, idk if in this context there could be other bootupd things).

Reply:
I liked the option being a bit abstract, since there may be eventually other things that will have to be done differently if the image uses `bootupd`... :thinking:

Reply:
Yup, my thinking exactly.